test/e2e/tests/settings-add-snap-account-toggle.spec.js

```diff
@@ -0,0 +1,52 @@
+const { strict: assert } = require('assert');
+const { withFixtures } = require('../helpers');
+const FixtureBuilder = require('../fixture-builder');
+
+describe('Add snap account experimental settings', function () {
+  it('switch "Enable Add snap account" to on', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder().build(),
+        title: this.test.title,
+        failOnConsoleError: false,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+        await driver.fill('#password', 'correct horse battery staple');
```

Comment:
This line and the next can be replaced by `await unlockWallet(driver);` going forward